src/pages/Newpost/index.tsx

```diff
@@ -1,41 +1,84 @@
 import Header from '@/components/header';
 import { Flex, Button, Image, Textarea, Box, Text, Input } from '@chakra-ui/react';
 import Musseuk from '@/assets/images/musseuk_semicolon.png';
+import Musseuk2 from '@/assets/images/musseuk_labtop.png';
+import Musseuk3 from '@/assets/images/musseuk_hood.png';
+import Musseuk4 from '@/assets/images/musseuk_heart.png';
+import Musseuk5 from '@/assets/images/musseuk_default.png';
 import { useNavigate } from 'react-router-dom';
 
 const NewPost = () => {
   const navigate = useNavigate();
   return (
     <>
-      <Box h="100vh" bgColor="color(display-p3 0.9765 0.9765 0.9569);">
-        <Box p="2rem" margin="auto" w="34%" textAlign="center">
+      <Box bgColor="color(display-p3 0.9765 0.9765 0.9569);">
+        <Box p="2rem" margin="auto" w="42%" textAlign="center">
           <Text marginBottom="1rem" textAlign="left" fontSize="1.5rem" color="black">
             머쓱이 이름
           </Text>
-          <Input marginBottom="2rem" placeholder="머쓱이 이름을 작성해 주세요!" size="md" />
+          <Input
+            fontSize="1.2rem"
+            height="4rem"
+            bgColor="white"
+            borderRadius="0.3rem"
+            marginBottom="2rem"
+            placeholder="머쓱이 이름을 작성해 주세요!"
+            size="md"
+          />
           <Text marginBottom="1rem" textAlign="left" fontSize="1.5rem" color="black">
             머쓱이 테마
           </Text>
-          <Box margin="auto" border="1px" h="35rem" p={4}>
-            <Image src={Musseuk} alt="Dan Abramov" />
+          <Box margin="auto" h="31rem" p={4}>
+            <Image src={Musseuk} alt="Musseuk" />
+          </Box>
+          <Box cursor="pointer" bgColor="white" width="auto" borderRadius="10px" border="1px" borderColor="#D4D8CA">
+            <Flex justify="center" align="center">
+              <Box _hover={{ border: '0.3rem solid #72D988', borderRadius: '10px' }} width="50rem" height="auto">
```

Comment:
P1:
아하! 요소가 hover 될 때마다 border 영역이 새롭게 생겨서 들쑥날쑥한 현상이 생겼던거였군요! 😮
기본 border 사이즈를 지정해주면 어느정도 완화가 되는 것 같아요!

```suggestion
              <Box
                  border="0.3rem solid transparent"
                  _hover={{ border: '0.3rem solid #72D988', borderRadius: '10px' }}
                  width="50rem"
                  height="auto">
                  <Image p="0.5rem 0 0.5rem 0" objectFit="fill" marginTop="0.7rem" src={Musseuk} alt="Musseuk" />
              </Box>
```